2019-Spring-WashU.md

```diff
@@ -400,11 +396,19 @@ Most class periods will be split between lecture/discussion about Open Source So
     * Topic: Open Source and IoT
 
 ### Mon May 6
+* Check-in:
```

Comment:
We could simplify this a lot by just saying, "Project is delivered to professors for evaluation"